tbd.schema.json

```diff
@@ -315,6 +322,27 @@
         "surfaces"
       ]
     },
+    "SubSurface": {
+      "description": "Suburface(s) U-factors (optional)",
```

Comment:
This is strictly to alloy users to set NRFC-rated U-factors (without having to rely on an E+ run to retrieve them).